docs/src/examples/components/Chat/Playground.tsx

```diff
@@ -0,0 +1,2 @@
+// Playground is not supported by this component
+export default null
```

Comment:
Can't we just check if this file is not present? This seems really awkward.. Or at least let's create Empty playground component, or something like that..

Reply:
We have:
- autogenerated playground (no file)
- we have a generated playground (file is present)
- no playground at all

Can you suggest a better option for the last item?...

I believe that once we will create missing playgrounds or improve their generation third option will not be required anymore. But I don't have more simple idea for now.

Reply:
I agree that we should not specify this using `null`. How about `NoPlayground` or `DisablePlayground` constant? Then we can also get rid of all the associated comments.

Reply:
It can be option
- we have a generated playground (file is present)

and it's just going to render an empty playground, you can even create this as @mnajdova suggests

Reply:
We agreed with @mnajdova on blacklist, is sounds like the best option